src/components/CheckedLabel/CheckedLabel.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {forwardRef} from 'react';
+import {block} from '../utils/cn';
+import {Props} from './types';
+
+import './CheckedLabel.scss';
+
+const b = block('checked-label');
+
+export const CheckedLabel = forwardRef<HTMLLabelElement, Props>(
```

Comment:
I would name the component as `ControlLabel`, checked word is too specific

Reply:
Not sure that this component will fit regular TextInput